WordPress/Sniffs/WP/I18nSniff.php

```diff
@@ -163,16 +166,17 @@ public function process( PHP_CodeSniffer_File $phpcs_file, $stack_ptr ) {
 		}
 
 		$func_open_paren_token = $phpcs_file->findNext( T_WHITESPACE, ( $stack_ptr + 1 ), null, true );
-		if ( false === $func_open_paren_token || T_OPEN_PARENTHESIS !== $tokens[ $func_open_paren_token ]['code'] ) {
+		if ( false === $func_open_paren_token || T_OPEN_PARENTHESIS !== $this->tokens[ $func_open_paren_token ]['code'] ) {
 			 return;
 		}
 
 		$arguments_tokens = array();
 		$argument_tokens  = array();
+		$tokens           = $this->tokens;
```

Comment:
Yes as line 202/206 overrides the local version of `$tokens`. See: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/827/files#diff-df68b36623d243f82931be87d069e1efL202